docs/standard/serialization/system-text-json-supported-collection-types.md

```diff
@@ -0,0 +1,293 @@
+---
+title: "Supported collection types in System.Text.Json"
+description: "Learn which collection types are supported for serialization by the APIs in the System.Text.Json namespace."
+ms.date: 01/04/2021
+no-loc: [System.Text.Json]
+zone_pivot_groups: dotnet-version
+helpviewer_keywords:
+  - "JSON serialization"
+  - "serializing objects"
+  - "serialization"
+  - "objects, serializing"
+---
+
+# Supported collection types in System.Text.Json
+
+This article gives an overview of which collections are supported for serialization and deserialization. <xref:System.Text.Json.JsonSerializer?displayProperty=nameWithType> supports a collection type for serialization if it:
+
+* Derives from <xref:System.Collections.IEnumerable>.
+* Contains elements that are serializable.
+
+The serializer calls the <xref:System.Collections.IEnumerable.GetEnumerator> method, and writes the elements.
+
+Deserialization is more complicated and is not supported for some collection types.
+
+## System.Collections namespace
```

Comment:
Maybe give an intro here to what the subsections are? The following sections are organized by namespace and show which types are supported for serialization and deserialization...